rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/OperatorMeteringLabels.java

```diff
@@ -0,0 +1,32 @@
+package io.syndesis.qe.rest.tests.steps;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.syndesis.qe.utils.OpenShiftUtils;
+import io.syndesis.qe.utils.TestUtils;
+
+import java.util.List;
+import java.util.Map;
+
+import cucumber.api.java.en.Then;
+import io.fabric8.kubernetes.api.model.Pod;
+
+public class OperatorMeteringLabels {
+    /**
+     * Checks whether pods contains com.redhat metering labels.
+     * It is a feature for Fuse Online product, therefore check runs only in case of the productized build.
+     */
+    @Then("verify new RedHat metering labels")
+    public void checkRedhatLabels() {
+        if (TestUtils.getSyndesisVersion().contains("redhat")) {
```

Comment:
You can use @prod test annotation (https://github.com/syndesisio/syndesis-qe/blob/1.10.x/rest-tests/src/test/java/io/syndesis/qe/RestTestHooks.java#L38) instead of if statement in order to run the test only with the prod builds.